### PR TITLE
[FIX] migration scripts for 13.0

### PR DESCRIPTION
**Pull request by etobella**
I found some errors when migrating to 13.0 using OpenUpgrade

### CODE REVIEW

**Comment by etobella** [comment]
@pedrobaeza

File: sale_commission_formula/migrations/13.0.1.0.0/pre-migration.py [hunk 1]
```diff
@@ -1,13 +0,0 @@
-# Copyright 2020 NextERP Romania SRL
```
Comment (pedrobaeza) [inline sale_commission_formula/migrations/13.0.1.0.0/pre-migration.py]: You have forgotten to add the post-migration file.

File: sale_commission/migrations/13.0.1.0.0/post-migration.py [hunk 2]
```diff
@@ -8,3 +8,36 @@ def migrate(env, version):
...
         env.cr, "sale_commission", "migrations/13.0.1.0.0/noupdate_changes.xml"
     )
+    openupgrade.logged_query(
+        env.cr,
+        """
+        UPDATE account_invoice_line_agent aila
+        SET object_id = aml.id
+        FROM account_move_line aml
+        WHERE aml.old_invoice_line_id = aila.{}
+    """.format(
```
Comment (pedrobaeza) [inline sale_commission/migrations/13.0.1.0.0/post-migration.py]: Please use `sql.SQL` and `sql.Identifier` for proper SQL build.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

File: sale_commission/migrations/13.0.1.0.0/post-migration.py (hunk 2 above)
Comment (etobella) [reply sale_commission/migrations/13.0.1.0.0/post-migration.py]: Sure!

**Comment by etobella** [comment]
@pedrobaeza Comments attended :wink:

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**